ros/src/waypoint_updater/waypnt_updater.py

```diff
@@ -485,18 +497,22 @@ def set_waypoints_velocity(self):
                                  lookahead_wps)
             else:
                 # reduce car velocity to 0.0 stoping before lights
+                rospy.loginfo("now start to slowdown")
```

Comment:
It's not an issue, but this prints out on every loop during the slowdown state.  It could be only printed if self.state != 'slowdown' or the text could be changed to "now slowing down".